### DISCUSSION TITLE
Closing UI on macOS issues

### DISCUSSION THREAD

**Discussion by levens-github** [Chia-Network/chia-blockchain#1409] [Q&A]
I closed the Chia app on macOS but I still see chia running and my drives are still grinding away, I waited a few minutes and then opened Chia and even though it knows my plot directories it shows no plotting. When you close the UI it says it will stop plotting and farming but it seems to keep running?

**Comment by levens-github** [comment]
I am running the GUI v1.0 on macOS 10.14.6 on iMac 5K

I rebooted and when I run chia GUI it shows no plots, no farming, nothing. When I add another plot it adds to the GUI and starts but nothing I do is finding my existing plots, none had finished, but I had 4 that have been plotting all day long. My plots are on a Pegasus32 DAS via thunderbolt and is faster than my local SSD. 

I tried Refresh Plots from GUI but nothing happens.

**Reply by levens-github** [reply]
OK so it seems if you close UI or stop before a plot completes it is gone, however it seems to leave behind all of the plot files. How do you clean this up without removing the completed plot? I screwed up and made 7 plots in one folder and both temp and complete were pointing to the same location. How can I remove the dead plots and files, keep the finished plot and modify my setup so that I have a separate temp and completed folder?

**Reply by zachlankton** [reply]
Unfortunately if you closed the gui before the plots are finished then you are only left with temp files which cannot be used to resume the plot.  If you have an actaul *.plot file in the directory and it is in fact the full size of the plot you were trying to create then you got lucky and still probably have a good plot file.  Copy this plot file to a folder/directory that you would like to store your plots in and then add that folder using the gui.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
